lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -20,6 +20,10 @@
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE TypeOperators #-}
 
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+-- Needed for (Buildable SlotNo)
+-- TODO#1901: How do we want to deal with the orphan (Buildable SlotNo) instance?
```

Comment:
I would add an `Cardano.Orphans` module or something like that, where it would be acceptable to have `-fno-warn-orphans`.

Reply:
👍  Done